src/touch/gestures.c

```diff
@@ -44,7 +46,7 @@ extern volatile uint8_t measurement_done_touch;
 
 /********************************** STATE **********************************/
 
-static const uint16_t LONG_TOUCH = 500;
+static const uint16_t LONG_TOUCH = 500 * 2;
```

Comment:
...1000? Also this will affect the BB02 behaviour!

Reply:
ack